importer/main.py

```diff
@@ -250,15 +250,38 @@ def organization_extras(resource, payload_string):
     return payload_string
 
 
+def identify_coding_object_index(array, current_system):
+    for index, value in enumerate(array):
+        list_of_systems = value["coding"][0]["system"]
+        if current_system in list_of_systems:
+            return index
+
+
+def check_parent_admin_level(locationParentId):
+    base_url = get_base_url()
+    resource_url = "/".join([base_url, "Location", locationParentId])
+    response = handle_request("GET", "", resource_url)
+    obj = json.loads(response[0])
+    response_type = obj["type"]
+    current_system = "administrative-level"
+    index = identify_coding_object_index(response_type, current_system)
+    if index >= 0:
```

Comment:
can we handle the case where the system is not in the list and "none" is returned? 
NoneTye cannot be compared or ">=" to 0